eahub/localgroups/views.py

```diff
@@ -116,3 +118,15 @@ def report_group_inactive(request, slug):
         ''' Thank you, we have received your report. Our admin team will send you an email once they have looked into it. ''',
     )
     return redirect('/group/{}'.format(group.slug))
+
+
+def report_abuse(request, slug):
```

Comment:
This should be a [`FormView`](https://docs.djangoproject.com/en/2.2/ref/class-based-views/generic-editing/#formview).

Reply:
implemented